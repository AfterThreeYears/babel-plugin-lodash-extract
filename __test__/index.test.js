const babel = require('babel-core');
const plugin = require('../');

var example = `
import _ from 'lodash'
import isArray from 'lodash/isArray'
import { toString } from 'lodash'

isArray([])
_.add(1, 2)
toString(1)
_.add(1, 2)
isArray([1])
`;

it('works', () => {
  const {code} = babel.transform(example, {plugins: [plugin]});
  expect(code).toMatchSnapshot();
});