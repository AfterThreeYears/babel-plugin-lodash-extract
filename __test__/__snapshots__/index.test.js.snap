// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"
import isArray from 'lodash/isArray';
import toString from 'lodash/toString';


isArray([]);
import add from 'lodash/add';
add(1, 2)

toString(1);
add(1, 2)

isArray([1]);"
`;
